scripts/inventory/ovirt4.py

```diff
@@ -106,13 +106,13 @@ def create_connection():
     config_path = os.environ.get('OVIRT_INI_PATH', default_path)
 
     # Create parser and add ovirt section if it doesn't exist:
-    config = configparser.SafeConfigParser(
+    config = configparser.ConfigParser(
```

Comment:
```suggestion
    config = configparser.ConfigParser(
```
For Python 2, it should still use `SafeConfigParser`.

Reply:
@abikouo ^

Reply:
yes will do it :)